packages/cloud-cognitive/src/components/CreateModal/CreateModal.stories.js

```diff
@@ -0,0 +1,271 @@
+      {children}
+    </CreateModal>
+  );
+};
+
+const TemplateWithStateManager = ({
+  storyInitiallyOpen = false,
+  story,
+  children,
+  ...args
+}) => {
+  const [open, setOpen] = useState(storyInitiallyOpen);
+
+  return (
+    <>
+      <Button onClick={() => setOpen(true)}>Open {story?.storyName}</Button>
+      <style>{`.${pkg.prefix}-create-modal { opacity: 0 }`};</style>
+      <CreateModal {...args} open={open} onClose={() => setOpen(false)}>
+        {children}
+      </CreateModal>
+    </>
+  );
+};
+
+const TemplateWithFormValidation = ({
+  storyInitiallyOpen = false,
+  story,
+  ...args
+}) => {
+  const [open, setOpen] = useState(storyInitiallyOpen);
+  const [textInput, setTextInput] = useState('');
+  const [invalid, setInvalid] = useState(false);
+
+  return (
+    <>
+      <Button onClick={() => setOpen(true)}>Open {story?.storyName}</Button>
+      <style>{`.${pkg.prefix}-create-modal { opacity: 0 }`};</style>
+      <CreateModal
+        {...args}
+        open={open}
+        onClose={() => setOpen(false)}
+        disableSubmit={textInput.length === 0 ? true : false}
+        primaryFocus=".bx--text-input">
+        <TextInput
+          id="1"
+          key="form-field-1"
+          labelText="Text input label"
+          placeholder="Placeholder"
+          onChange={(e) => {
+            setTextInput(e.target.value);
+            setInvalid(false);
+          }}
+          onBlur={() => {
+            textInput.length === 0 && setInvalid(true);
+          }}
+          invalid={invalid}
+          invalidText="This is a required field"
+        />
+        <TextInput
+          id="2"
+          key="form-field-2"
+          labelText="Text input label (optional)"
+          placeholder="Placeholder"
+        />
+        <TextInput
+          id="3"
+          key="form-field-3"
+          labelText="Text input label (optional)"
+          placeholder="Placeholder"
+        />
+        <RadioButtonGroup
+          legendText="Radio button legend text goes here"
+          name="radio-button-group"
+          defaultSelected="radio-1">
+          <RadioButton labelText="Radio-1" value="radio-1" id="radio-1" />
+          <RadioButton labelText="Radio-2" value="radio-2" id="radio-2" />
+          <RadioButton labelText="Radio-3" value="radio-3" id="radio-3" />
+        </RadioButtonGroup>
+      </CreateModal>
+    </>
+  );
+};
+
+const defaultProps = {
+  title: 'Title',
+  subtitle: 'Your subtitle text will appear here',
+  description:
+    'This is example description text that will appear here in your modal ',
+  primaryFocus: '.bx--text-input',
+};
+
+Template.propTypes = {
+  children: PropTypes.node,
+};
+
+TemplateWithStateManager.propTypes = {
+  story: PropTypes.object,
+  storyInitiallyOpen: PropTypes.bool,
+  ...CreateModal.propTypes,
+};
+
+TemplateWithFormValidation.propTypes = {
+  story: PropTypes.object,
+  storyInitiallyOpen: PropTypes.bool,
+  ...CreateModal.propTypes,
+};
```

Comment:
You don't have to give the storybook stories prop types because they are not public-facing and just adds more code.